--- origsrc/lualdap-1.1.0/Makefile	2007-03-13 17:07:33.000000000 -0500
+++ src/lualdap-1.1.0/Makefile	2008-02-13 20:15:42.437500000 -0600
@@ -22,7 +22,6 @@
 install: src/$(LIBNAME)
 	mkdir -p $(LUA_LIBDIR)
 	cp src/$(LIBNAME) $(LUA_LIBDIR)
-	cd $(LUA_LIBDIR); ln -f -s $(LIBNAME) $T.so
 
 clean:
 	rm -f $(OBJS) src/$(LIBNAME)
--- origsrc/lualdap-1.1.0/config	2006-07-23 20:42:06.000000000 -0500
+++ src/lualdap-1.1.0/config	2008-02-13 20:13:03.968750000 -0600
@@ -14,11 +14,11 @@
 
 # Lua version number (first and second digits of target version)
 LUA_VERSION_NUM= 500
-LIBNAME= $T.so.$V
+LIBNAME= $T.dll
 COMPAT_DIR= ../compat/src
 
 # Compilation parameters
-WARN= -O2 -Wall -fPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -ansi
+WARN= -O2 -Wall -DPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -ansi
 INCS= -I$(LUA_INC) -I$(OPENLDAP_INC) -I$(COMPAT_DIR)
 CFLAGS= $(WARN) $(INCS)
 CC= gcc
